### ISSUE TITLE
Removing ServerHeader

### ISSUE THREAD

**Issue by thefudgeishot** [CodeCrafter47/BungeeTabListPlus#301]
<img width="1440" alt="screen shot 2018-01-05 at 9 24 41 am" src="https://user-images.githubusercontent.com/33798627/34592024-456c96da-f1fc-11e7-804f-2b83edecb448.png">
Im trying to remove this server header and replace it with the player list. So basically a full tab of players. Is this possible???

It would be nice if this is fixed soon 
thanks
Fudge

**Comment by CodeCrafter47** [comment]
Yes that is possible. You probably want to use the `!players` instead of the `!players_by_server` component. I can recommend you to read the wiki, especially the following page: <https://github.com/CodeCrafter47/BungeeTabListPlus/wiki/Configuration-Basics-Part-7---Fixed-Size-Tablist---Adding-Players>, but you might also want to have a look at the other wiki pages to learn more about configuring the plugin.

**Comment by thefudgeishot** [comment]
<img width="1440" alt="screen shot 2018-01-06 at 9 45 07 am" src="https://user-images.githubusercontent.com/33798627/34635142-5b864248-f2c6-11e7-85c9-bec683ff0dfa.png">
by doing what the forum said to do i end up with my tablist looking like this. 
Just in case here's the config bit
<img width="1023" alt="screen shot 2018-01-06 at 9 48 19 am" src="https://user-images.githubusercontent.com/33798627/34635177-cd0b0d0e-f2c6-11e7-87bd-2d2748720ca6.png">

**Comment by CodeCrafter47** [comment]
Try removing lines 156 to 167.

**Comment by thefudgeishot** [comment]
![image](https://user-images.githubusercontent.com/33798627/35478763-35d26b5a-0421-11e8-8c00-713de77476d4.png)



Same issue i get the broken tablist

![](https://user-images.githubusercontent.com/33798627/34635142-5b864248-f2c6-11e7-85c9-bec683ff0dfa.png)

**Comment by thefudgeishot** [comment]
@CodeCrafter47

**Comment by CodeCrafter47** [comment]
Now you're again using `!players_by_server` instead of `!players`. Decide what you want to do.

Also if it's broken, you can have a look at the proxy log file, there's an error message telling you what's wrong.
